common/source/docs/common-spracingh7-extreme.rst

```diff
@@ -0,0 +1,155 @@
+.. _common-spracingh7-extreme:
+
+===================
+SPRacing H7 Extreme
+===================
+
+The SPRacingH7 Extreme is a flight controller produced by `Seriously Pro Racing <http://www.seriouslypro.com/>`__ .
+
+.. image:: ../../../images/SPracing-H7-extreme.jpg
+     :target: ../_images/SPracing-H7-extreme.jpg
+     :width: 240px
+
+.. image:: ../../../images/SPracing-H7-extreme2.jpg
+     :target: ../_images/SPracing-H7-extreme2.jpg
+     :width: 240px
+
+Specifications
+==============
+
+-  **Processor**
+
+   -  STM32F750  ARM 32-bit processor running at 400 MHz
+   -  OSD: AT7456E 
+   -  16MB Flash for data logging, WTE has microSD card reader instead of flash chip
```

Comment:
Not sure what WTE is, but we only support the version with a microSD card reader

Reply:
So no 16MB dataflash

Reply:
cut and paste error....and it has a 1 gigabit (128MByte) external NAND instead of the 16MB one I had...fixed